src/components/App.jsx

```diff
@@ -16,7 +45,19 @@ function App() {
       </div>
       <div className="row">
         <div className="col">
```

Comment:
why does this div have a col class? doesn't it default to a col if its the only one in the row?